jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -347,7 +345,7 @@ def _plot_mask(self):
         if 'mask' in _data.components and self.display_mask:
 
             # If a mask was already plotted, remove it.
-            if hasattr(self,'mask_trace_pointer') and \
+            if hasattr(self, 'mask_trace_pointer') and \
                     self.mask_trace_pointer is not None:
```

Comment:
I am surprised it didn't complain about `\`.
```suggestion
            if (hasattr(self, 'mask_trace_pointer') and
                    self.mask_trace_pointer is not None):
```